web/Views/Admin/TenantList.cshtml

```diff
@@ -11,6 +11,13 @@
         <i class="bi bi-arrow-90deg-down"></i> Filter
     </a>
 </div>
+<div class="col-8">
+    @using (Html.BeginForm("SortTenantsByLocation", "Tenant", FormMethod.Get))
+    {
+        <input class="form-control" id="postcode" name="postcode" required placeholder="Enter postcode to sort by location" pattern="([Gg][Ii][Rr] 0[Aa]{2})|((([A-Za-z][0-9]{1,2})|(([A-Za-z][A-Ha-hJ-Yj-y][0-9]{1,2})|(([A-Za-z][0-9][A-Za-z])|([A-Za-z][A-Ha-hJ-Yj-y][0-9][A-Za-z]?))))\s?[0-9][A-Za-z]{2})"/>
+        <button type="submit" class="btn custom-btn-primary"> Sort </button>

```

Comment:
It would be better to use padding to space out the text in this button instead of using spaces

Reply:
I'll be honest - I copied this button from another page, I don't even know if it needs spacing out.